src/main/java/com/minelittlepony/client/transform/PonyTransformation.java

```diff
@@ -29,15 +29,52 @@ public void transform(PonyModel<?> model, BodyPart part, MatrixStack stack) {
                     if (model.getAttributes().isCrouching) stack.translate(-0.03F, 0.03F, 0.13F);
                     break;
                 case HEAD:
-                    if (model.getAttributes().isLyingDown) stack.translate(-0.05F, -0.05F, 0);
-                    if (model.getAttributes().isCrouching) stack.translate(0, 0.1F, -0);
+                    if (model.getAttributes().isLyingDown) stack.translate(0, -0.05F, 0);
+                    if (model.getAttributes().isCrouching) stack.translate(0, 0.1F, 0);
                     break;
                 case BACK:
                     translateForRider(stack);
                     break;
                 default:
             }
         }
+    },
+	STOCKY(SizePreset.STOCKY, 0, 3F, 0.75F) {
+        @Override
+        public void transform(PonyModel<?> model, BodyPart part, MatrixStack stack) {
+            if (model.getAttributes().isSwimming) stack.translate(0, -0.3F, 0);
+            if (model.getAttributes().isCrouching) stack.translate(0, -0.2F, 0);
+            if (model.getAttributes().isLyingDown) stack.translate(0, -0.77F, 0.1F);
+            if (model.getAttributes().isSleeping) stack.translate(0, 0.16F, 0);
+            if (model.getAttributes().isSitting) stack.translate(0, -0.25F, -0.2F);
+
+            switch (part) {
+                case NECK:
+                    stack.translate(-0.015F, -0.05F, -0.04F);
+					stack.scale(1.4F, 1, 1.2F);
+                    if (model.getAttributes().isCrouching) stack.translate(-0.025F, 0.03F, 0.15F);
+                    if (model.getAttributes().isSitting) stack.translate(0, 0, 0.03F);
+                    break;
+                case HEAD:
+                    if (model.getAttributes().isLyingDown) stack.translate(0, -0.05F, 0);
+                    if (model.getAttributes().isCrouching) stack.translate(0, 0.1F, 0);
+                    break;
+				case BODY:
```

Comment:
Can you correct your indentions to be the same as the rest of the file please? This is probably an issue with how your IDE is configured

Reply:
oh sure! i can do that
not sure why it's like that... i shall look into it!!